packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -226,401 +272,185 @@ class _ReorderableListContent extends StatefulWidget {
-
-  // This controls the entrance of the dragging widget into a new place.
-  late AnimationController _entranceController;
-
-  // This controls the 'ghost' of the dragging widget, which is left behind
-  // where the widget used to be.
-  late AnimationController _ghostController;
-
-  // The member of widget.children currently being dragged.
-  //
-  // Null if no drag is underway.
-  Key? _dragging;
-
-  // The last computed size of the feedback widget being dragged.
-  Size? _draggingFeedbackSize;
-
-  // The location that the dragging widget occupied before it started to drag.
-  int _dragStartIndex = 0;
-
-  // The index that the dragging widget most recently left.
-  // This is used to show an animation of the widget's position.
-  int _ghostIndex = 0;
-
-  // The index that the dragging widget currently occupies.
-  int _currentIndex = 0;
-
-  // The widget to move the dragging widget too after the current index.
-  int _nextIndex = 0;
-
-  // Whether or not we are currently scrolling this view to show a widget.
-  bool _scrolling = false;
-
-  double get _dropAreaExtent {
-    if (_draggingFeedbackSize == null) {
-      return _defaultDropAreaExtent;
-    }
-    final double dropAreaWithoutMargin;
-    switch (widget.scrollDirection) {
-      case Axis.horizontal:
-        dropAreaWithoutMargin = _draggingFeedbackSize!.width;
-        break;
-      case Axis.vertical:
-        dropAreaWithoutMargin = _draggingFeedbackSize!.height;
-        break;
+class _ReorderableListContentState extends State<_ReorderableListContent> {
+  Widget _wrapWithSemantics(Widget child, int index) {
+    void reorder(int startIndex, int endIndex) {
+      if (startIndex != endIndex)
+        widget.onReorder(startIndex, endIndex);
     }
-    return dropAreaWithoutMargin;
-  }
-
-  @override
-  void initState() {
-    super.initState();
-    _entranceController = AnimationController(vsync: this, duration: _reorderAnimationDuration);
-    _ghostController = AnimationController(vsync: this, duration: _reorderAnimationDuration);
-    _entranceController.addStatusListener(_onEntranceStatusChanged);
-  }
-
-  @override
-  void didChangeDependencies() {
-    _scrollController = widget.scrollController ?? PrimaryScrollController.of(context) ?? ScrollController();
-    super.didChangeDependencies();
-  }
-
-  @override
-  void dispose() {
-    _entranceController.dispose();
-    _ghostController.dispose();
-    super.dispose();
-  }
 
-  // Animates the droppable space from _currentIndex to _nextIndex.
-  void _requestAnimationToNextIndex() {
-    if (_entranceController.isCompleted) {
-      _ghostIndex = _currentIndex;
-      if (_nextIndex == _currentIndex) {
-        return;
+    // First, determine which semantics actions apply.
+    final Map<CustomSemanticsAction, VoidCallback> semanticsActions = <CustomSemanticsAction, VoidCallback>{};
+
+    // Create the appropriate semantics actions.
+    void moveToStart() => reorder(index, 0);
```

Comment:
Good question. I think so, as the indices all work the same in reverse, just the display is inverted. So moving to 0 is still moving to the start of the list, but the start of the list is displayed at the bottom instead of the top.